App/src/components/NewCalendar.js

```diff
@@ -96,14 +96,25 @@ function NewCalendar({
   forecast, // the forecast data
   users, // the list of users to show
   selectedUsers = [], // the list of users to highlight
-  selectedDate = null // the date to highlight those users
+  selectedDate = null, // the date to highlight those users
+  shiftsClicked
```

Comment:
Instead of adding a new field you could pass `selectedUsers` and `selectedDate` to know workshift highlight.